wdio.conf.js

```diff
@@ -19,8 +19,7 @@ const config = {
   baseUrl: `http://${localIP.address()}:${webpackPort}`,
   specs,
 
-  // Travis only has 1 browser instace, set maxInstances to 1 to prevent timeouts
-  maxInstances: process.env.CI ? 1 : wdioConf.config.maxInstances,
+  maxInstances: 1,
```

Comment:
can we put this change in toolkit?

Reply:
Yes. should i make the change there? & remove maxInstances from our config?

Reply:
Yeah, lets do that. If we want to handle max instances again after this bug is fixed, we should handle it from toolkit.